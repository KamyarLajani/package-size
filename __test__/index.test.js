const getSizes = require('../lib')

jest.setTimeout(10000)

test('main', async () => {
  const data = await getSizes('vue@2.0.0')
  expect(data).toMatchSnapshot()
})
