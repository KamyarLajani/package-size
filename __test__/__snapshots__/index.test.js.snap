// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main 1`] = `
Object {
  "gzipped": 27051,
  "minified": 74283,
  "name": "vue@2.0.0",
  "size": 189575,
  "versionedName": "vue@2.0.0",
}
`;
